src/inmanta/app.py

```diff
@@ -75,9 +77,32 @@
 class MultiLineFormatter(colorlog.ColoredFormatter):
     """Multi-line formatter."""
 
+    def __init__(
+        self,
+        fmt: typing.Optional[str] = None,
+        datefmt: typing.Optional[str] = None,
+        style: _FormatStyle = "%",
+        log_colors: typing.Optional[LogColors] = None,
+        reset: bool = True,
+        secondary_log_colors: typing.Optional[SecondaryLogColors] = None,
+        validate: bool = True,
+        stream: typing.Optional[typing.IO] = None,
+        no_color: bool = False,
+        force_color: bool = False,
+        defaults: typing.Optional[typing.Mapping[str, typing.Any]] = None,
+        timed: bool = False,
+    ):
+        super().__init__(
+            fmt, datefmt, style, log_colors, reset, secondary_log_colors, validate, stream, no_color, force_color, defaults
+        )
+        self.timed = timed
+
     def get_header_length(self, record):
         """Get the header length of a given record."""
-        header = super().format(
+        log_format = "%(asctime)s " if self.timed else ""
+        log_format += "%(name)-25s%(levelname)-8s%(message)s"
```

Comment:
I don't really get this. This is not stable, is it? How about using `ColoredFormatter(log_colors=False, reset=False, force_color=False, ...)` where `...` represents this instance's attributes?

Reply:
I created a Formatter that matches the one currently used (https://github.com/inmanta/inmanta-core/blob/df161b561b1d880c65db6d8ff5c496242e25e27b/src/inmanta/app.py#L742)
but removing the color parts. I'll try your approach

Reply:
My main issue with it is that the class seems to be generic but then there's this hardcoded assumption in there. It would be great if we could get rid of that. Related to your link: I now just noticed that we only use this multiline logger when `timed` is true. Was this a conscious decision? If so, could you elaborate?

Reply:
we don't? we use the `MultiLineFormatter` when `_is_on_tty` is true. 
timed just prepends the time to the log_format

Reply:
Right, my bad. Same question though, why only on a tty? What's the alternative scenario? Is this the compile reports on the dashboard? I can seem why we might not want indentation there? Could you perhaps add a comment as to why we use a plain `Formatter`?

Reply:
I was already like that when it was a `colorlog.ColoredFormatter` So I think it is more about the color codes than the indentations. Maybe I could call the `MultiLineFormatter` in the else without colors but I don't know what this if/else had as purpose neither

Reply:
@arnaudsjs do you know what [this](https://github.com/inmanta/inmanta-core/blob/df161b561b1d880c65db6d8ff5c496242e25e27b/src/inmanta/app.py#L752) is about?

Reply:
I've added the `MultiLineFormatter` to the else clause but without colors for now

Reply:
Great, Arnaud and I concluded that we'd like to go with that for now. I would suggest not merging it until after we have branched off for the releases though, so we have some time to let this mature.

Reply:
I'll wait with the review request until then :)